### DISCUSSION TITLE
WSA 2309 Update

### DISCUSSION THREAD

**Discussion by jaholme** [microsoft/WSA#430] [Announcements]
# What's New

We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2309.40000.2.0) includes improvements to platform reliability and functionality improvements.

## Release Notes

- Platform reliability improvements
- Updated Chromium WebView to version 117
- Allow .cer files to be shared to Android
- Graphics improvements
-  Android Geocoder APIs ([Geocoder  |  Android Developers](https://developer.android.com/reference/android/location/Geocoder)) provide valid data for all applications. Be sure to check the terms of use for using Bing Maps data. ([Microsoft Software License Terms Microsoft Windows Subsystem for Android™ - Microsoft Support](https://support.microsoft.com/en-us/windows/microsoft-software-license-terms-microsoft-windows-subsystem-for-android-cf8dfb03-ba62-4daa-b7f3-e2cb18f968ad))
- Android 13 Platform Updates

# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by Andarwinux** [comment]
The graphics improvements is a "Vulkan 1.3.0" driver with only 36 extensions?

![image](https://github.com/microsoft/WSA/assets/144242044/7e7e0488-6cd1-49ee-b771-23ba8e9c4cef)
![image](https://github.com/microsoft/WSA/assets/144242044/3462a715-524a-42a3-a6ff-7bc630a1336b)

**Reply by WesleyVanNeck** [reply]
did anyting change in opengl-es

**Reply by Andarwinux** [reply]
No changes, just some additional placeholder extensions to indicate gfxstream capabilities.

**Comment by s1204IT** [comment]
@jaholme
When will the 2309 kernel source be released?

https://github.com/microsoft/WSA-Linux-Kernel/tags
I can only find sources up to 2308.

**Reply by jaholme** [reply]
The kernel used in the 2309 release is published under this tag:
https://github.com/microsoft/WSA-Linux-Kernel/releases/tag/android-lts%2Flatte-2%2F5.15.104.4

**Reply by s1204IT** [reply]
You (guys) were finally able to publish it xD
thx.
